src/setup/plugin-multi-server-use/index.md

```diff
@@ -0,0 +1,32 @@
+# How to use MerginMaps plugin with multiple servers
+
+There is a default [MerginMaps server](https://public.cloudmergin.co.uk) which is preconfigured in QGIS plugin. But there might be some cases you might want to use some custom server (e.g. you host it yourself). We will cover two typical use cases
+
+- migration from one server to another
+- using multiple servers simultaneously
+
+## Non-standard configuration
+
+Let's start with setting of custom Mergin server. It can be done in a configuration dialog
```

Comment:
https://dev.merginmaps.com/docs/misc/write-docs/#naming-things

in plain text use https://github.com/MerginMaps/docs/tree/main/src/.vuepress/components